docs/README-encryption.md

```diff
@@ -63,19 +63,22 @@ TDX_ENC_KEY_BACKEND:forcevariable = "tpm"
 
 Make sure to use the `forcevariable` override, so your configuration takes precedence over the default one.
 
-A few additional variables are available to customize the behavior of the data-at-rest encryption feature. Here is the complete list:
+A few additional variables are available to customize the behavior of the data-at-rest encryption feature. Here is list of the most important ones (for the full list refer to `tdx-encrypted.bbclass`):
```

Comment:
Is there any other user-facing variable not documented here?

Reply:
As I mentioned, the ones related to data preservation.

Reply:
Done

Reply:
Yeah, you are right. Makes sense to document it. But does not need to be done as part of this PR. Thanks for pointing that out in the documentation.